org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -500,9 +500,10 @@ public static BuildConfiguration getBuildConfiguration(Path rootFolder, boolean
 		File javaHome = getJavaHome(preferences);
 		File gradleUserHome = getGradleUserHomeFile();
 		List<String> gradleArguments = new ArrayList<>();
-		if (noDaemon) {
-			gradleArguments.add("--no-daemon");
-		}
+		// https://github.com/microsoft/vscode-gradle/issues/1519
+		//if (noDaemon) {
+		//	gradleArguments.add("--no-daemon");
+		//}
```

Comment:
Just to confirm, Even with this option gone, the Gradle project importer still creates its own separate process. Where is that defined ?

Reply:
See https://github.com/gradle/gradle/blob/f7cb2605b8ead8c23aaa1c4f881f0c5c3a03e239/platforms/core-runtime/launcher/src/main/java/org/gradle/tooling/internal/provider/ProviderStartParameterConverter.java#L58